stripe/src/main/java/com/stripe/android/paymentsheet/ui/BasePaymentSheetActivity.kt

```diff
@@ -56,6 +56,10 @@ internal abstract class BasePaymentSheetActivity<ResultType> : AppCompatActivity
                 0f
             }
 
+            // When using commit on the fragments, the fragment transaction happens
+            // at some later time.  In order to get an accurate backstack count
+            // we need to make sure the transactions have completed.
+            supportFragmentManager.executePendingTransactions()
```

Comment:
Here is the executePendingTransactions I mentioned

Reply:
sgtm. Is there any downside of using `executePendingTransactions`?